client/app/telecom/pack/migration/building-details/telecom-pack-migration-building-details.controller.js

```diff
@@ -114,6 +114,23 @@ angular.module('managerApp').controller('TelecomPackMigrationBuildingDetailsCtrl
     return stairsModel;
   }
 
+  /**
+   * Set default stairs model for empty stairs
+   */
+  defaultStairsModel() {
+    const stairsModel = {};
```

Comment:
You can merge declarations, and return it directly :)

Reply:
done